File: find-the-number-of-good-pairs-i.ts
{
    function numberOfPairs(nums1: number[], nums2: number[], k: number): number {
        let c = 0
        
        for (let i = 0; i < nums1.length; i++) {
            const num_1 = nums1[i]
            for (let j = 0; j < nums2.length; j++) {
                const num_2 = nums2[j] * k

                if (num_1 % num_2 === 0) 
                    c++
          
            }

        }

        return c
    }

    console.log(numberOfPairs([1,3,4], [1,3,4], 1))
}

File: find-the-number-of-good-pairs-i.rs
pub fn number_of_pairs(nums1: Vec<i32>, nums2: Vec<i32>, k: i32) -> i32 {
    let mut c = 0;

    for num_1 in &nums1 {

        for num_2 in &nums2 {

            if num_1 % (num_2 * k) == 0 {
                c += 1;
            }

        }
    }

    return c;   
}